ArduPlane/quadplane.cpp

```diff
@@ -2895,6 +2921,32 @@ void QuadPlane::vtol_position_controller(void)
     }
 }
 
+void QuadPlane::assign_tilt_to_fwd_thr(void) {
+    if (is_positive(q_fwd_thr_gain)) {
```

Comment:
I'm getting a bit picky now, so feel free to ignore. My preference when there is a lot of stuff in a if statement like this and then a else is to do the smaller bit first in this case we can do it and then return.

 ```suggestion
    if (!is_positive(q_fwd_thr_gain)) {
        q_fwd_throttle = 0.0f;
        return;
    }    
```

Then its very clear we don't do anything for none positive gains, and the core of the function is one level less indented.

Reply:
Done